roles/manage-confluence-space/tasks/copy_confluence_attachments.yml

```diff
@@ -1,10 +1,10 @@
 ---
 - name: Get attachment data
   uri:
-    url: '{{ confluence_space_source_url }}/wiki/rest/api/content/{{ confluence_content_ids.key }}/child/attachment'
+    url: '{{ confluence.source.url }}/rest/api/content/{{ confluence_content_ids.key }}/child/attachment'
```

Comment:
This seems to be reverting the changes made in #203 - i.e.: the addition of `/wiki` - please check/correct.